### PR TITLE
Adding roadmap points

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Moving over a few roadmap points, if you have these already, just remove or tick that one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
-# Medium Upload
+# Atom Medium
 
-Write your Medium stories in Atom like a hacker and upload them to directly Medium as a draft or a public story.
+Write your Medium stories in Atom like a hacker and publish them to directly Medium as a draft or a public story.
 
-![info-bar](https://raw.githubusercontent.com/ericadamski/atom-medium-upload/master/assets/info-bar.png)
+![info-bar](https://raw.githubusercontent.com/ericadamski/atom-atom-medium/master/assets/info-bar.png)
```
Comment (ericadamski) [inline README.md]: this should be renamed to `atom-medium`

File: README.md [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
 
-# Usage
+## Usage
 
-![menu item](https://raw.githubusercontent.com/ericadamski/atom-medium-upload/master/assets/menu-item.png)
+![menu item](https://raw.githubusercontent.com/ericadamski/atom-atom-medium/master/assets/menu-item.png)
```
Comment (ericadamski) [inline README.md]: Same with this one

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## 0.1.3
-* [Stop using reserved JavaScript keyword `package`](https://github.com/ericadamski/atom-medium-upload/pull/9)
+* [Stop using reserved JavaScript keyword `package`](https://github.com/ericadamski/atom-atom-medium/pull/9)
```
Comment (ericadamski) [inline CHANGELOG.md]: this should be renamed to `atom-medium`

**Review by ericadamski — CHANGES_REQUESTED** [review]
@ChrisChinchilla sorry about never looking through this! I want to do a lot of refactoring of this code and really get this package to a state that more people can maintain it so I am going to push all these PRs through. If you cannot devote the time to changes on these just let me know and I can fix them all to get them merged!

**Comment by ChrisChinchilla** [comment]
To be honest @ericadamski I hit a lot of the authentication issues that Jared hit in recent issues and got a bit stuck. I will attempt to pick up the ball again if you're still keen, but for now, just go ahead with making the changes you want.

**Comment by ericadamski** [comment]
@ChrisChinchilla I think I need to spend some time to re-write this package because coming back to it the logic is all over the place 😜 I think the issue #14 my bring some attention to the authentication issues as we comb through the code

**Comment by ericadamski** [comment]
This PR is out-of-date but the road map items still relevant have been added.

**Pull request closed without merging**